docs/application/native/api/iot-headed/5.5/group__Evas__Font__Group.html

```diff
@@ -279,7 +282,7 @@
 </div>
 <div class="memdoc">
 <p>Reinitialize FontConfig. If FontConfig has to be reinitialized according to changes of system environments (e.g. Changing font config files), it will be useful.</p>
-<dl class="since"><dt><b>Since:</b></dt><dd>1.14 </dd></dl>
+<dl class="since"><dt><b>Since:</b></dt><dd>1.14  3.0f(2.3.1) </dd></dl>
```

Comment:
Would you please check this update?

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/